scripts/run-para-test-single.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/sh
+
+# If you are in the tests folder, go back to the root of the repository
+FOLDER="${PWD##*/}"
+if [[ "$FOLDER" == "tests" ]]; then
```

Comment:
Shouldn't you test if in the scripts folder instead ?

Reply:
These scripts are related to the tests, I had initially placed them in the tests/ folder then seeing that this folder did not contain any bash script I then moved them in /script

Reply:
I think all the scripts are expected to run from the root folder, but if someone tries to run them, it is will be from the script folder (where they are located).
We could improve this by doing a better check (like git commands are doing for ex).

Reply:
I replaced "tests" by "scripts" :)